pkgs/by-name/repath-studio/package.nix

```
@@ -0,0 +1,201 @@
+{
+  lib,
+  stdenv,
+
+  fetchFromGitHub,
+  fetchNpmDeps,
+  electron,
+  chromium,
+  clojure,
+  npmHooks,
+  nodejs,
+
+  writeShellScriptBin,
+  copyDesktopItems,
+  makeDesktopItem,
+  makeWrapper,
+  replaceVars,
+
+  vulkan-loader,
+}:
+stdenv.mkDerivation (finalAttrs: {
```

No need to add `nodejs` and `npmHooks` hooks as well

```suggestion
buildNpmPackage (finalAttrs: {
```